registrations/permissions.py

```diff
@@ -217,3 +217,8 @@ def has_object_permission(
         return request.user.is_superuser or request.user.is_financial_admin_of(
             obj.publisher
         )
+
+
+class WebStoreWebhookPermission(permissions.BasePermission):
+    def has_permission(self, request: Request, view: APIView) -> bool:
+        return request.user.is_authenticated
```

Comment:
There is [rest_framework.permissions.IsAuthenticated](https://www.django-rest-framework.org/api-guide/permissions/#isauthenticated) that does the same so I don't think this is needed.

Reply:
Ah yes, the same permission class that I removed some time ago and apparently had my memory erased after that. Changed as suggested.